ribasim_qgis/core/nodes.py

```diff
@@ -164,6 +147,10 @@ def set_editor_widget(self) -> None:
         # Calling during new_layer doesn't have any effect...
         pass
 
+    def load_style(self):
+        fn = STYLE_DIR / f"{self.input_type().replace(' / ', '_')}Style.qml"
+        self.layer.loadNamedStyle(str(fn))
```

Comment:
Is this loaded only in the layer, so only for the QGIS session? That means the plugin currently doesn't put the Ribasim default style in the GeoPackage for new tables right?